scss/modules/_feedback.scss

```diff
@@ -19,42 +19,49 @@
   }
 
   textarea {
-    height: 8rem;
+    height: 7rem;
     margin-bottom: 2rem;
+
+    &:last-of-type {
+      margin-bottom: 1rem;
+    }
   }
 
   a {
     border-color: $cream;
   }
 
+  .label {
+    &:first-of-type {
+      margin-top: 2rem;
+    }
+  }
+
   .feedback__close {
     position: absolute;
     right: 1rem;
     top: 1rem;
   }
 
-  &[aria-hidden="true"] {
-  }
-
   .message {
     background-color: transparent;
-    border: none;
+    border: 0;
     margin-top: 0;
   }
 
   @include media($lg) {
-    @include transition(bottom .5s);  
-    bottom: 0;
+    @include transition(top .5s);
+    height: 95vh;
     left: auto;
     overflow: auto;
-    padding: 3rem;
+    padding: 3rem 3rem 0 3rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `3rem 3rem 0` instead of `3rem 3rem 0 3rem`